code/datums/wires/syndicatebomb.dm

```diff
@@ -42,9 +46,12 @@
 				playsound(B, 'sound/machines/click.ogg', 30, 1)
 				B.active = TRUE
 				B.update_icon()
+			if(B.delayedlittle)
+				holder.visible_message("<span class='notice'>\icon[B] Nothing happens.</span>")
 			else
 				holder.visible_message("<span class='notice'>\icon[B] The bomb seems to hesitate for a moment.</span>")
-				B.timer += 5
+				B.timer += 10
+				B.delayedlittle = TRUE

```

Comment:
It's alright like that. You can add an if(active) for bigdelay if you want but I don't think anyone will mind if it's not there.